### ISSUE TITLE
allow adding new nodes to snippets manager (feature request)

### ISSUE THREAD

**Issue by verajosemanuel** [rstudio/rstudio#3789]
Sometimes I use Rstudio for any other language than the expected R or Rmarkdown, and the creation of nodes in snippet manager (Edit Snippet Menu) for the needed languages instead of being limited to the offered (R, C, Rmd, TeX, Js, html, css, sql, java, python and stan) would be great.

**Comment by ronblum** [comment]
@verajosemanuel Thanks for the feature suggestion! We'll take a look at it as we continue to develop new versions of RStudio.

**Comment by GegznaV** [comment]
I agree, that there should be a possibility to use snippets in more languages. E.g., a possibility to use YAML snippets would be great.

**Comment by GegznaV** [comment]
Not implemented...

**Comment by GegznaV** [comment]
Not implemented...

**Comment by GegznaV** [comment]
Is there a plan to implement this soon?
